src/main/java/org/broadinstitute/hellbender/utils/config/ConfigUtils.java

```diff
@@ -0,0 +1,332 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import org.aeonbits.owner.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A class to contain configuration utility methods.
+ * Created by jonn on 7/19/17.
+ */
+public class ConfigUtils {
+
+    static private final Logger logger = LogManager.getLogger(ConfigUtils.class);
+
+    // This class just has static methods to help with configuration, so no need for this constructor.
+    private ConfigUtils() {}
+
+    /**
+     * Whether we have already set the config factory variable defaults.
+     */
+    private static boolean hasSetConfigFactoryVariableDefaults = false;
+
+    /**
+     * Sets the {@link org.aeonbits.owner.ConfigFactory} variables so that it knows about
+     * the variable paths for config files.
+     */
+    public static final void setConfigFactoryVariableDefaults() {
+
+        // You only need to do this once.
+
+        if ( !hasSetConfigFactoryVariableDefaults ) {
+
+            ArrayList<String> propertyNames = new ArrayList<>();
```

Comment:
It's good to make even your instance variables final when they can be, you can configure intellij to warn you when it's possible.